website/docs/reference/transforms/lua.md

```diff
@@ -274,13 +396,11 @@ set to an invalid value, a message will be logged and the field will be dropped.
 
 
 [docs.configuration#environment-variables]: /docs/setup/configuration/#environment-variables
-[docs.data-model.log#schema]: /docs/about/data-model/log/#schema
 [docs.data-model.log]: /docs/about/data-model/log/
+[docs.data-model.metric]: /docs/about/data-model/metric/
```

Comment:
The inconsistent naming here is unfortunate. :(

Reply:
I agree, it makes sense to use snake case everywhere.